### PR TITLE
Feature/orm

### PR DESCRIPTION
**Pull request by fisher60**
Implement ORM for basic recipe/ingredient models utilizing sqlite. I have also created a sample endpoint to get all recipe objects.

### CODE REVIEW

File: Controllers/RecipeController.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        _context = context;
+    }
+
+    [HttpGet(Name = "GetRecipes")]
+    public IList<Recipe> GetRecipes()
+    {
+        using (var context = new RecipleaseContext()){
+            var recipes = context.Recipes.ToList();
+            return recipes;
+        }
```
Comment (GDWR) [inline Controllers/RecipeController.cs:25]: do we need to make a new context? Can't we use the `_context` from the Controller?

File: Models/Recipe.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+namespace Reciplease;
+
+public class Recipe 
+{
+    public int Id { get; set; }
+    public int Servings { get; set; }
+
+    [Required]
+    public string Name { get; set; }
```
Comment (GDWR) [inline Models/Recipe.cs:11]: Why is this field required? and the others are not?

File: README.md [hunk 3]
```diff
@@ -36,4 +36,13 @@ INSERT INTO recipe_ingredient (recipe_id, ingredient) VALUES (1, 'Chicken');
 INSERT INTO recipe_ingredient (recipe_id, ingredient) VALUES (1, 'Mushroom');
 INSERT INTO recipe_ingredient (recipe_id, ingredient) VALUES (2, 'Mushroom');
 INSERT INTO recipe_ingredient (recipe_id, ingredient) VALUES (3, 'Orange');
+```
+
+### Database Migrations
+Currently the `reciplease.db` file is created in the local app directory. `C:\Users\<user>\AppData\Local` on windows.
```
Comment (GDWR) [inline README.md:42]: Update documentation, also remove the RAW SQL test

**Review by GDWR — CHANGES_REQUESTED** [review]

File: Models/Recipe.cs (hunk 2 above)
Comment (fisher60) [reply Models/Recipe.cs:11]: Int's are implicitly non-nullable, strings are by default nullable, so they must be explicitly marked as not nullable if they are required.

**Review by GDWR — APPROVED** [review]

**Comment by GDWR** [comment]
closed #1

**Pull request merged**